examples/react-hmr/rolldown.config.js

```diff
@@ -21,4 +21,5 @@ export default defineConfig({
   output: {
     format: 'app',
   },
+  dev: true,
```

Comment:
Is it better to use `hmr` or `watch`? The `dev` option is somewhat ambiguous IMO.

Reply:
It's only a simple option to show case. We need to review the API if we merge it into main branch.